pom.xml

```diff
@@ -45,6 +45,10 @@
     xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
     <parent>
+        <!--@todo #909:30min jcabi-parent used by Netbout is pretty outdated,
```

Comment:
@dmzaytsev No, hang on, ignore me, it is part of the original ticket. Weird though, I think it should be separated.